### DISCUSSION TITLE
Need to use it with line command, I need an example please (help) -> Cuting a clip at time X and Y

### DISCUSSION THREAD

**Discussion by APISeeker** [mifi/lossless-cut#1876] [Q&A]
Hello
Could someone give me the exact command line I should write to "cut a given audio at a CERTAIN time"? without losing quality of course.
I tried to do some research but I am not sure I undertood everything.

**Comment by mifi** [comment]
I think you’re better off using ffmpeg directly for this

**Reply by APISeeker** [reply]
Hello @mifi, will it be "loseless" and as good as the software loselesscut then?

**Reply by mifi** [reply]
yes
